src/Atoms/Loader/Loader.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Container = styled.div`
+  width: 75px;
+  height: 15px;
+  margin: 20px auto;
+
+  & > * {
+    animation: opacity 1s infinite;
+    width: 15px;
+    height: 15px;
+    margin: 0 5px;
+    -webkit-border-radius: 100%;
+    -moz-border-radius: 100%;
+    border-radius: 100%;
+    background-color: ${({ theme }) => theme.colors.primary};
+    float: left;
+  }
+
+  @keyframes opacity {
+    0%, 100% { opacity: 1; }
+    60% { opacity: 0; }
+  }
+
+  @-webkit-keyframes opacity {

```

Comment:
The CSS rules are automatically vendor prefixed, styled-components takes care of that for you!

Reply:
o nice